### ISSUE TITLE
technology radar doesn't utilize description

### ISSUE THREAD

**Issue by johnnyiller** [backstage/backstage#4493]
When I follow the documentation for adding data to the technology radar it lists url and description in the data structure.  It would seem logical that this data would be shown to users somewhere within the interface.

## Expected Behavior

Description data is shown to the user in some fashion

## Current Behavior

nothing, data is not being utilized

## Possible Solution

Update the technology radar plugin to display the description when the user hovers over a blip or the name of the technology or both.

## Steps to Reproduce

if you just visit the default tech radar and click around you'll see it does not utilize the description field in any way.

## Your Environment

- NodeJS Version (v12):
- Operating System and Version (e.g. Ubuntu 14.04):
- Browser Information: chrome

**Comment by johnnyiller** [comment]
I'll also, mention I'd be happy to work on this issue.

**Comment by adamdmharvey** [comment]
Awesome - welcome the contrib, you got it @johnnyiller !

**Comment by adamdmharvey** [comment]
I'll add in, when I've used the Tech Radar, I've never really had enough info on the elements.  I like the hover. I also wonder whether or not a separate tab, that maybe presents each element in the text stack with a description and maybe a link out to much more info.  Allows the user to USE that plugin to really LEARN about the stack, as opposed to just the visual.

All in all, a way for me to allow my new hires to learn more about the stack and where to get more info on it. (maybe even linking to a TechDocs releavent page that elaborates further, which could just be done via URL as it's most simple)  Your tool tip idea for sure helps elevate the info if it's not used today!

**Comment by johnnyiller** [comment]
@adamdmharvey agreed.  For this round, I'm just going to try to get the information that the README implies should work to appear somewhere in the UI in a reasonable way, but thin, the more we can make the process one of exploration the better since that seems to be the intent of a technology radar in the first place.

**Comment by johnnyiller** [comment]
@adamdmharvey I assume you'll see the PR for this issue, but not super clear how it works.  This work is done for my purposes but perhaps you have some other changes that make sense. If so I'd be happy to add them.

**Comment by niallmccullagh** [comment]
Hi @johnnyiller/@adamdmharvey,

I love the dialog but it got me thinking around the meaning of the description within the timeline. 

Should it be used to describe the timeline event? 
Maybe we need a top level description field that would we used for the dialog?
The dialog could be expanded later to show the timeline entries.

**Comment by johnnyiller** [comment]
@niallmccullagh thats a great point.  My intention here was to leave the existing data structure, but it probably does make sense to explain the technology separate from the decision to transition the technology.  If @adamdmharvey is agreeable to that direction I'd be cool with it.
